lia-app-uikit/HomeView.swift

```diff
@@ -76,4 +76,27 @@ class HomeView: UIViewController {
         let ltvc = ListTableViewController()
         self.navigationController?.pushViewController(ltvc, animated: true)
     }
+    
+    // Creating the time based greeting text function.
+    func greetingLogic() -> String {
+        let hour = Calendar.current.component(.hour, from: Date())
+        
+        let NEW_DAY = 0
+        let NOON = 12
+        let SUNSET = 18
+        let MIDNIGHT = 24
+        
+        var greetingText = "Welcome!"
+        switch hour {
+        case NEW_DAY..<NOON:
+            greetingText = "Good Morning!"
+        case NOON..<SUNSET:
+            greetingText = "Good Afternoon!"
+        case SUNSET..<MIDNIGHT:
+            greetingText = "Good Evening!"
+        default:
+            _ = "Welcome!"
```

Comment:
Perhaps the default message should be like Hello otherwise the app will read "Welcome Welcome to Lia Dog Breed App"?